view/theme/frio/js/theme.js

```diff
@@ -609,132 +600,30 @@ String.prototype.rtrim = function() {
 	return trimmed;
 };
 
-/**
- * Scroll the screen to the item element whose id is provided, then highlights it
- *
- * Note: jquery.color.js is required
- *
- * @param {string} elementId The item element id
- * @returns {undefined}
- */
-function scrollToItem(elementId) {
-	if (typeof elementId === "undefined") {
+// Scroll to a specific item and highlight it
+// Note: jquery.color.js is needed
+function scrollToItem(itemID) {
+	if( typeof itemID === "undefined")
 		return;
-	}
 
-	var $el = $(document.getElementById(elementId));
+	var elm = $('#'+itemID);
```

Comment:
We couldn't use jQuery in this function. Because jQuery will not work with hubzilla guid (@ and dots in the guid)